### DISCUSSION TITLE
API Not working

### DISCUSSION THREAD

**Discussion by Monon-sr** [Zeddxx/ani-fire#36] [General]
Bro API not working... Please help me... Please... Please... Please..😍

**Comment by Zeddxx** [comment]
@Monon-sr Which api?

**Reply by anitummy** [reply]
Aniwatch.to API... Please bro help me... 😢

**Reply by anitummy** [reply]
I mean Primary URL....

**Reply by anitummy** [reply]
Just take a look... https://anitummy.vercel.app/. Please help me 😭... Which API i should use... Please tell me...

**Reply by Zeddxx** [reply]
@anitummy try to redeploy the api this time on render check if thats works or not.

**Reply by Zeddxx** [reply]
As it should work check how the owner gave steps to deploy the api.

**Reply by anitummy** [reply]
Okay... I'm trying then...

**Reply by anitummy** [reply]
Bro it's not working... Please help me...

**Reply by Zeddxx** [reply]
It is working man, give me the screenshot of primary-url

**Reply by anitummy** [reply]
![IMG_20240331_191904](https://github.com/Zeddxx/ani-fire/assets/165540602/f42f2fdb-5484-4f89-80a5-1a2f249a7fd1)

**Reply by Zeddxx** [reply]
Your api have some issues, you tried the vercel?
P.s. onrender is slow your api is giving required details

**Reply by anitummy** [reply]
Will you please help me... Can you give me an primary url... Please... Please.   Please... Please... please...😭😭😭

**Reply by anitummy** [reply]
brother how can i set AUTH_SECRET= please tell me....

**Reply by Zeddxx** [reply]
@anitummy search for generate secret online you will get a generator for that if not then type any random strings.
